server/tools/docker-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
why not ``set -e`` in a separate line?
Also we probably want ``set -eou pipefail``

Reply:
Yes, that would also work. About the other options I am not sure it that would break, I did not test with them. But I agree that it would be very nice to have them.

Reply:
`-o pipefail` would only forward errors trough pipes so if the first command fails the entire line fails. Very good combo with `-e`.
`-u` errors when unset variables are accessed. This normally needs some `${VAR:-}` to fall back to empty and not error.

Reply:
I know what they do and I gave them both a thought, but I also wanted to minimize the changes of how the script behaves. Adding `-u` would need a lot more extensive testing of it for example.

Reply:
> why not `set -e` in a separate line?
> Also we probably want `set -eou pipefail`

+1 for changing this into the `set -eou pipefail` form (from Python Zen, when it should fail, fail early and with as much verbosity as possible)